source/base/bounding_box.cc

```diff
@@ -55,42 +52,53 @@ BoundingBox<spacedim, Number>::merge_with(
                  other_bbox.boundary_points.first[i]);
       this->boundary_points.second[i] =
         std::max(this->boundary_points.second[i],
-                 other_bbox.boundary_points.second[i]);
+                    other_bbox.boundary_points.second[i]);
     }
```

Comment:
This is a separate question if I understand you correctly, *independent* of the new function. Would you be willing to submit a separate pull request for this?

Reply:
I am currently having problems with my clang-format integration. This is related to this issue. Sorry, I will remove this change.